actors.md

```diff
@@ -152,17 +152,12 @@ type AccountActor struct {
 
 ## Storage Market Actor
 
-The storage market actor is the central point for the Filecoin storage market. It is responsible for registering new miners to the system, and maintaining the power table. The Filecoin storage market is a singleton that lives at a specific well-known address.
+The storage market actor is the central point for the Filecoin storage market. It is responsible for registering new miners to the system, and maintaining the power table. The Filecoin storage market is a singleton, defined in terms of a particular state in the Filecoin state tree and that lives at a specific well-known address.
 
-```go
-type StorageMarketActor struct {
```

Comment:
the actors state should not be removed, thats a thing that needs to be clearly defined.

Reply:
Happy to add back in, but I thought the interface in storage-market spec did that well enough. What am I not understanding? It also seemed more fleshed out than what was in actors (which I interpreted as stale).

What can I do to fix?

Reply:
the contents of actors.md is not stale, its the absolute truth of how the network works. If something seems off to you there, please point it out.

Reply:
mmmh I was just going off of the fact that `totalStorage` was its only field. Therefore, thought we might be missing things (ie stale) given the interface for storage-market in storage-market.md..

Probably issue on my end.